doc/index.rst

```diff
@@ -34,9 +34,24 @@ There are three different ways which you can use Cube Browser:
 Installation
 ____________
 
-Conda is great
+In the following pages you will find downloads of Jupyter notebooks containing Cube Browser operations.
+To be able to use these notebooks, you will need to configure your environment correctly.
+The most efficient and clean method to do this is to use a Conda environment (please see `Anaconda website <https://anaconda.org/>`_ for help and information).
```

Comment:
`Anaconda website` feels some what tautological... I guess you don't have to mask every URL with link text :wink: